docs/csharp/language-reference/keywords/index.md

```diff
@@ -76,6 +76,14 @@ Keywords are predefined, reserved identifiers that have special meanings to the
 |[var](../../../csharp/language-reference/keywords/var.md)|[where (generic type constraint)](../../../csharp/language-reference/keywords/where-generic-type-constraint.md)|[where (query clause)](../../../csharp/language-reference/keywords/where-clause.md)|  
 |[yield](../../../csharp/language-reference/keywords/yield.md)||  
   
+## Single-character keywords
+
+Single-character keywords are symbols that have a special meaning to the compiler. The `@` symbol is the verbatim identifier character; it can be used to distinguish an identifier (such as a variable named `@for`) from a C# keyword, as well as to indicate that the characters in a string are interpreted literally. The `$` symbol is the interpolated string character; it identifies the literal string that follows as an interpolated string.
+
+|||  
+|---|---|---|  
+|[@](../../../csharp/language-reference/keywords/verbatim.md)|[$](../../../csharp/language-reference/keywords/interpolated.md)||
```

Comment:
I agree. Unless there are other elements to add, I think it's probably best to present this as a bulleted list -- I'll make the correction.